static-assets/components/cstudio-forms/forms-engine.js

```diff
@@ -2787,6 +2787,7 @@ const initializeCStudioForms = () => {
                 }
               });
           });
+          form.sections.push(formSection);
```

Comment:
The issue happens when rendering only some fields in the form (not the full form)
When validating the form, what the forms-engine does is to check against the list of sections rendered. In the path where selected fields are rendered, we were not setting the form section into the sections variable of the form, so when validating, sections = [] and no validations were performed.